WtWSMS/common/casus_belli_types/T4N_punative_war.txt

```diff
@@ -0,0 +1,239 @@
+﻿punative_war_cb = {
```

Comment:
Typo here and in the file title, no?

Reply:
Will need to reinstall the spell-checker.